enterprise/frontend/src/metabase-enterprise/auth/components/SettingsJWTForm/SettingsJWTForm.tsx

```diff
@@ -102,13 +102,7 @@ export const SettingsJWTForm = ({
           <FormSection title={"Server Settings"}>
             <Stack spacing="md">
               <FormTextInput {...fields["jwt-identity-provider-uri"]} />
-              <FormSecretKey
-                {...fields["jwt-shared-secret"]}
-                confirmation={{
-                  header: t`Regenerate JWT signing key?`,
-                  dialog: t`This will cause existing tokens to stop working until the identity provider is updated with the new key.`,
-                }}
-              />
+              <FormSecretKey {...fields["jwt-shared-secret"]} />
```

Comment:
`confirmation` fields were hard-coded and this component is only used once so it doesn't need to be customised, so I moved the text to the component directly where it's used